src/content/app/genome-browser/components/browser-nav-button/BrowserNavButton.tsx

```diff
@@ -15,7 +15,8 @@
  */
 
 import React, { memo } from 'react';
-import type { OutgoingAction } from '@ensembl/ensembl-genome-browser';
+import { OutgoingAction } from '@ensembl/ensembl-genome-browser';
```

Comment:
Why the change here?

Reply:
Probably missed this one during the merge

✅